src/Assignments/ProblemDetails.tsx

```diff
@@ -12,16 +12,23 @@ const INFINITE_MAX_ATTEMPT_VALUE = 0;
 interface ProblemDetailsProps {
     problem: ProblemObject;
     topic: TopicObject | null;
+    attemptsRemaining?: number;
 }
 
 export const ProblemDetails: React.FC<ProblemDetailsProps> = ({
     problem,
     topic,
+    attemptsRemaining,
 }) => {
-    const startDate = moment(topic?.startDate);
-    const endDate = moment(topic?.endDate);
-    const deadDate = moment(topic?.deadDate);
-    const solutionsMoment = moment(deadDate).add(1, 'days');
+    const isAssessment = !_.isNil(topic?.topicAssessmentInfo?.studentTopicAssessmentInfo?.[0]);
```

Comment:
This isn't strictly true, right? The true designator is what the topicTypeId is.

Reply:
Unless we handle that filtering on the backend (but it wouldn't hurt to be defensive against it on the frontend)

Reply:
you're right about this -- we should check topicTypeId *and* this is not nil, and change the name to `isVersionedAssessment` to be clear that we're checking for a version of the topic, which IS an assessment...